src/main/java/com/google/devtools/build/lib/packages/TargetUtils.java

```diff
@@ -28,12 +25,12 @@
 import com.google.devtools.build.lib.syntax.SkylarkDict;
 import com.google.devtools.build.lib.syntax.Type;
 import com.google.devtools.build.lib.util.Pair;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+
 import javax.annotation.Nullable;
+import java.util.*;
```

Comment:
for future: add the google-java-style in your IDE.

Reply:
fixed :)